src/main/java/io/logz/guice/jersey/JerseyServer.java

```diff
@@ -55,25 +58,28 @@ private void configureServer() {
             server.addConnector(connector);
         });
 
-        WebAppContext webAppContext = new WebAppContext();
-        webAppContext.setServer(server);
+        if (websocketConfiguration != null && "/".equals(jerseyConfiguration.getContextPath())) {
```

Comment:
We need to add contextPath param to websocket configuration and letting the user decide what is the desired context root for Jersey and for the websockets.
Then the check should be if websocketConfiguration not null and the context paths are not the same.